src/include/execution/vm/bytecode_handlers.h

```diff
@@ -606,6 +607,11 @@ VM_OP_HOT void OpForceBoolTruth(bool *result, terrier::execution::sql::BoolVal *
   *result = input->ForceTruth();
 }
 
+VM_OP_HOT void OpInitSqlNull(terrier::execution::sql::Val *result) {
+  *result = terrier::execution::sql::Val::Null();
+  result->is_null_ = true;
```

Comment:
You can remove this line since the `is_null` flag should have been set in `Val::Null()`.